### PR TITLE
Test

### PR DESCRIPTION
**Pull request by tdpearson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,25 +19,59 @@ DEVELOPMENTAL AT THIS POINT! This is not an offical release. Please experiment a
...
         $ make init
 
-3. Build and Deploy
+4. Build and Deploy on local system.
 
         $ make build
         $ make run
 
-4. API running http://localhost:8080
+5. API running http://localhost:8080
```
Comment (tdpearson) [inline README.md]: Should we remove the port 8080 from this?

File: README.md [hunk 2]
```diff
@@ -19,25 +19,59 @@ DEVELOPMENTAL AT THIS POINT! This is not an offical release. Please experiment a
...
+4. Initialize and Get TLS certificates from LetsEncrypt
+        
+        $ make init_certbot
+
+5. Build and Deploy on local system.
+
+        $ make build
+        $ make run_with_certbot
+
+6. API running https://{domain-name-of-server}:8080
```
Comment (tdpearson) [inline README.md]: Should we remove port 8080 from this?

File: dc_config/images/celery/startCeleryWorker [hunk 3]
```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import os
 from subprocess import call
 
 for itm in os.environ.get("CELERY_SOURCE").split(','):
-    call(['pip','install',itm])
+    call(['pip3','install',itm])
```
Comment (tdpearson) [inline dc_config/images/celery/startCeleryWorker:7]: Look into modifying to use `python3 -m pip install`

File: dc_config/nginx/config/letsencrypt.conf [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    server_tokens off;
+
+    ssl_certificate /etc/letsencrypt/live/test.cc.lib.ou.edu/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/test.cc.lib.ou.edu/privkey.pem;
+
+    ssl_buffer_size 8k;
+
+    ssl_dhparam /etc/ssl/certs/dhparam-2048.pem;
+
+    ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
```
Comment (tdpearson) [inline dc_config/nginx/config/letsencrypt.conf]: Confirm the need to support older TLS 1.0 and 1.1. If needed, keep this as is. Do we want to add TLS 1.3?

File: docker-compose.yml [hunk 5]
```diff
@@ -1,11 +1,25 @@
 version: "3.7"
 services:
+  cybercom_nginx:
+    image: nginx:1.17
+    expose:
+      - 80
+      - 443
```
Comment (tdpearson) [inline docker-compose.yml]: Do we need port 443 here? If not, remove expose and ports.

**Review by tdpearson — COMMENTED** [review]
Address comments

File: dc_config/nginx/config/letsencrypt.conf (hunk 4 above)
Comment (tdpearson) [reply dc_config/nginx/config/letsencrypt.conf]: Let's keep 1.0 and 1.1 for now. What is needed to support TLS 1.3 in Python 2.7?

File: docker-compose.yml (hunk 5 above)
Comment (Nithivarn-Reddy) [reply docker-compose.yml]: Removed 443 from both ports and expose.

File: dc_config/nginx/config/letsencrypt.conf (hunk 4 above)
Comment (Nithivarn-Reddy) [reply dc_config/nginx/config/letsencrypt.conf]: Added TLSv1.3 and its working fine.

File: dc_config/images/celery/startCeleryWorker (hunk 3 above)
Comment (Nithivarn-Reddy) [reply dc_config/images/celery/startCeleryWorker:7]: This way of doing is erroring out.

**Review by Nithivarn-Reddy — COMMENTED** [review]
Removed portno 8080

File: dc_config/images/celery/startCeleryWorker (hunk 3 above)
Comment (tdpearson) [reply dc_config/images/celery/startCeleryWorker:7]: Let's keep as is.

File: dc_config/nginx/config/letsencrypt.conf (hunk 4 above)
Comment (tdpearson) [reply dc_config/nginx/config/letsencrypt.conf]: Looks good

File: docker-compose.yml (hunk 5 above)
Comment (tdpearson) [reply docker-compose.yml]: Looks good

**Pull request merged**